### DISCUSSION TITLE
How to catch group tab order change event?

### DISCUSSION THREAD

**Discussion by turquoise8** [mathuo/dockview#565] [Q&A] [answered]
![image](https://github.com/mathuo/dockview/assets/83459683/c1ee047f-6dea-4bb8-83d4-c24aff0f0041)

When changing the places of panel4 and panel5 here for instance, the API does not trigger any kind of events. onDidLayoutChange does not trigger, or any other onDid... events. Therefore I can't save the order of these tabs. Is there a way to do this? Am I missing something?

Thanks in advance.

**Comment by mathuo** [comment, marked as answer]
Registed as enhancement in https://github.com/mathuo/dockview/issues/582
